os/arch/arm/src/amebad/amebad_nsc.c

```diff
@@ -30,6 +31,33 @@ extern volatile TZ_MemoryId_t tz_memory;
 /****************************************************************************
  * Context Management for RTL8721D AmebaD TrustZone
  ****************************************************************************/
+void TZ_LoadContext(uint32_t *reg)
```

Comment:
I think these are not board-specific. How about moving them into context.h like mpu?
```
static inline void TZ_LoadContext( )
{
 ....
}
```

Reply:
And could you follow our coding rule for new function you made?
TZ_LoadContext_s is defined in ARM so that we don't need to change that.

# Naming
[M12] No mixed-case, Use underscores('_') to separate words in an name.

[Example-Do]
```
#define TEST_VALUE	32
extern struct list_head_hash[TEST_VALUE];

int example_function(void)
{
    return(TEST_VALUE - 1);
}
```

[Example-Don't]
```
#define Test_Value	32
extern struct list_head_hash[Test_Value];

int ExampleFunction(void)
{
    return(Test_Value - 1);
}
```

Reply:
ok